docs/theory/diffusion/diffusion.rst

```diff
@@ -0,0 +1,713 @@
+            for _ in range(100)
+        ]
+        return tracks, params["diffusion_constant"]
+
+    methods = {
+        "cve single": lambda list_of_tracks: [t.estimate_diffusion("cve") for tracks in list_of_tracks for t in tracks],
+        "cve ensemble": lambda list_of_tracks: [t.ensemble_diffusion("cve") for t in list_of_tracks],
+    }
+
+    snrs = 10**np.arange(-1, 1.1, 0.125)
+    results, variances = test_estimation_methods(simulate_tracks, snrs, methods=methods)
+    for method, estimates in results.items():
+        plot_accuracy(snrs, estimates, variance=variances[method], label=method);
+
+    plt.ylim([-5, 6])
+
+.. image:: figures/cve_ensemble.png
+
+Here we simulated `10` tracks with the same `8` steps as in the case which performed so poorly before.
+As expected, the ensemble CVE results in a far lower variances (each estimate uses more information).
+The uncertainty estimate Pylake returns on the individual CVEs is a little conservative this time (vertical lines), but the ensemble uncertainty estimate is pretty accurate.
+
+If we can safely assume that the localization uncertainty is constant then it is possible to improve the diffusion estimates by computing an ensemble averaged localization error first and then using that estimate when determining the per-track CVE.
+Let's define a function that does just this and compare it to the single step CVE::
+
+    def cve_single_using_ensemble_loc_uncertainty(tracks):
+        ensemble_estimate = tracks.ensemble_diffusion("cve")
+        return tracks.estimate_diffusion(
+            method="cve",
+            # Plug in the ensemble localization uncertainty estimates
+            localization_variance=ensemble_estimate.localization_variance,
+            variance_of_localization_variance=ensemble_estimate.variance_of_localization_variance
+        )
+
+    methods = {
+        "cve single": lambda list_of_tracks: [t.estimate_diffusion("cve") for tracks in list_of_tracks for t in tracks],
+        "cve single + fixed variance": lambda list_of_tracks: [t for tracks in list_of_tracks for t in cve_single_using_ensemble_loc_uncertainty(tracks)],
+        "cve ensemble": lambda list_of_tracks: [t.ensemble_diffusion("cve") for t in list_of_tracks],
+    }
+
+    results, variances = test_estimation_methods(simulate_tracks, snrs, methods=methods)
+    for method, estimates in results.items():
+        plot_accuracy(snrs, estimates, variance=variances[method], label=method);
+
+    plt.ylim([-5, 6])
+
+.. image:: figures/cve_carryover.png
+
+As expected, the uncertainty lies somewhere between the ensemble estimate and the individual track estimate.
+Note however that we would be getting this precision on a per track basis.
+This means that we could still use this analysis to obtain a distribution of diffusion estimates and see if there are subgroups with different diffusion constants.
+
+MSD-based methods
+^^^^^^^^^^^^^^^^^
+
+For MSD based methods, this same procedure is not optimal.
```

Comment:
I'd either remove this sentence, or be more explicit what 'this procedure' is.

Reply:
Good point! I've changed it to: 
```
For MSD based methods, simply calculating a weighted average of per-track estimates is not optimal.
```